walkthroughs/week-4-libraries/maps/examples/hello-world/src/main/webapp/index.html

```diff
@@ -1,14 +1,14 @@
 <!DOCTYPE html>
 <html>
-<head>
-  <title>Hello Google Maps</title>
-  <script src="https://maps.googleapis.com/maps/api/js?key=YOUR_API_KEY"></script>
-  <script src="script.js"></script>
-  <link rel="stylesheet" href="style.css">
-</head>
-<body onload="createMap();">
-  <h1>Hello Google Maps</h1>
-  <div id="map"></div>
-  <p>This page shows a Google Map!</p>
-</body>
+  <head>
```

Comment:
For all these files, remove these extra spaces.

Reply:
That is a file I was given by them to look at. I did not change it.

Reply:
Hmm ok. The github diff shows that you did, but maybe its git being weird?